mock/cpp_mock_scenarios/src/speed_planning/request_speed_change_continuous_false.cpp

```diff
@@ -42,6 +42,23 @@ class RequestSpeedChangeScenario : public cpp_mock_scenarios::CppScenarioNode
   bool speed_reached;
   void onUpdate() override
   {
+    /**
+     * @brief checking linear acceleration
+     */
+    if (api_.getCurrentTime() != 0.0 && api_.getCurrentTime() <= 1.0) {
+      if (api_.getEntityStatus("ego").action_status.accel.linear.x != 10.0) {
+        stop(cpp_mock_scenarios::Result::FAILURE);
+      }
+    }
+    if (api_.getCurrentTime() >= 1.05) {
+      if (api_.getEntityStatus("ego").action_status.accel.linear.x > 3.0) {
+        stop(cpp_mock_scenarios::Result::FAILURE);
+      }
+    }
```

Comment:
Why not use `and` operator as follows?

```
    if (api_.getCurrentTime() != 0.0 && api_.getCurrentTime() <= 1.0 and
        api_.getEntityStatus("ego").action_status.accel.linear.x != 10.0) {
      stop(cpp_mock_scenarios::Result::FAILURE);
    }
    if (api_.getCurrentTime() >= 1.05 and
        api_.getEntityStatus("ego").action_status.accel.linear.x > 3.0) {
      stop(cpp_mock_scenarios::Result::FAILURE);
    }
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/752/commits/d3a526269739fc082ea990eeb99c1c8676501182